src/main/java/com/minecolonies/coremod/entity/ai/citizen/baker/Product.java

```diff
@@ -0,0 +1,206 @@
+package com.minecolonies.coremod.entity.ai.citizen.baker;
+
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import org.jetbrains.annotations.NotNull;
+
+public class Product
```

Comment:
Rename to BakingProduct or make it more generic. What ever you do, this is not very flexible.